js/templates/modals/settings/addressesForm.html

```diff
@@ -0,0 +1,83 @@
+<form class="box border padKids padStack clrS clrBr js-addAddressForm">
```

Comment:
Looks like some of the styling is off. The tabColumn element should have clrP and a border on the right (with clrBr), settingsTabFormWrapper should have clrS and the form should have clrP.